data/json/monstergroups/zombie_upgrades.json

```
@@ -33,6 +33,15 @@
       { "monster": "mon_zombie_thorny", "freq": 15, "cost_multiplier": 5 }
     ]
   },
+  {
+    "name": "GROUP_SKELETON_UPGRADE",
+    "type": "monstergroup",
+    "default": "mon_skeleton",
+    "monsters": [
+      { "monster": "mon_skeleton_brute", "freq": 80, "cost_multiplier": 2 },
+      { "monster": "mon_skeleton_brute", "freq": 80, "cost_multiplier": 2 }
```

I'm assuming it was meant to look like this instead of having an 84% chance to make a skeleton evolve back into another skeleton? That said, if the new skeleton mobs also already get injected into new monstergroups with start delays, then one or the other may be a touch redundant except for monstergroups which lack any skeletons (if any).
```suggestion
    "default": "mon_skeleton_brute",
    "monsters": [
      { "monster": "mon_skeleton_necro", "freq": 80, "cost_multiplier": 2 },
      { "monster": "mon_skeleton_master", "freq": 80, "cost_multiplier": 2 }
```

---

I sort of addressed this in the post:
> This change is sort of weird after the removal, but I figured would be useful for new skeleton variants in the future when we add or port any so left it in the PR.

I added default last second and the monstergroups don't seem to work how I thought they did, so I'm glad you caught that. Do we feel that it's worth leaving this monstergroup out for now then (I didn't intend for us to do necro/master spawns the old way, but rather follow the new way we spawn them), until any additional skeleton variants are added?